docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Vote Account Management
+---
+
+This page describes how to set up an on-chain _vote account_.   Creating a vote
+account is needed if you plan to run a validator node on Solana.
+
+All settings for a vote account can be configured at the time the account is
+created, or they can be set after the account is created while the corresponding
+validator is running.
+
+### Vote Account Address
+
+A vote account is created at an address that is either the public key of a
+keypair file, or at a derived address from the authorized voter's public key.
```

Comment:
A derived address doesn't technically have to be based off either authority. It can be any ole keypair.  Perhaps that technicality doesn't need communicated here though?

Reply:
I can leave the technicality out, but the CLI description is actually rather confusing to me, looking at it again:
```
        --seed <STRING>
            Seed for address generation; if specified, the resulting account will be at a derived address of the VOTE
            ACCOUNT pubkey
```
That's the only place we use the term VOTE ACCOUNT in the help text.  I assume this is referring to the vote account keypair positional arg?  In which case my description here about deriving from the vote auth pubkey is just wrong anyway.